packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -296,6 +337,47 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 								}
 							) }
 						</p>
+						<div>
+							<FocusedLaunchSummaryItem
+								readOnly={ hasPaidDomain || selectedPaidDomain }
+								onClick={ () => defaultFreePlan && onSetPlan( defaultFreePlan ) }
+								isSelected={ ! ( hasPaidDomain || selectedPaidDomain ) && selectedPlan?.isFree }
+							>
+								<LeadingContentSide label={ defaultFreePlan?.titleWithPlanSuffix } />
+								<TrailingContentSide
+									price={ __( 'Free', __i18n_text_domain__ ) }
+									warningNote={
+										hasPaidDomain || selectedPaidDomain
+											? __( 'Not available with your domain selection', __i18n_text_domain__ )
+											: ''
+									}
+								/>
+							</FocusedLaunchSummaryItem>
+							<FocusedLaunchSummaryItem
+								onClick={ () => defaultPaidPlan && onSetPlan( defaultPaidPlan ) }
+								isSelected={ selectedPlan?.storeSlug === defaultPaidPlan?.storeSlug }
+							>
+								<LeadingContentSide
+									label={ defaultPaidPlan?.titleWithPlanSuffix }
+									badgeText={
+										defaultPaidPlan?.isPopular ? __( 'Popular', __i18n_text_domain__ ) : ''
+									}
+								/>
+								<TrailingContentSide
+									price={
+										<>
+											<span>{ defaultPaidPlan && planPrices[ defaultPaidPlan?.storeSlug ] }</span>
```

Comment:
Not a blocker, but I think that we should refactor the code to handle better the lack of data (ie. when it's loading, or when it's failing).

I opened #47410 to keep track of the loading aspect